### PR TITLE
Support PaymentInstrumentUpdate

### PR DESCRIPTION
**Pull request by phuonghuynh**

### CODE REVIEW

File: tests/Finix/ScenariosTest.php [hunk 1]
```diff
@@ -181,9 +185,53 @@ public function testSettlement()
...
     public function testDispute() {
         $transfer = $this->testDebitTransfer();
-        $disputePage = Dispute::getPagination($transfer->getHref("disputes"));
+        $disputePage = Dispute::getPagination($transfer);
         $dispute = $disputePage->items[0];
         $file = $dispute->uploadEvidence($this->receiptImage);
         $this->assertEquals($file->dispute, $dispute->id);
     }
+
+    public function testCreateAndFetchInstrumentUpdate()
```
Comment (phuonghuynh) [inline tests/Finix/ScenariosTest.php]: Create and Fetch InstrumentUpdate

File: src/Finix/Resource.php [hunk 2]
```diff
@@ -218,4 +224,16 @@ public function verifyOn(Verification $verification)
         $verifyLink = $this->resource->getLink("verifications")->getHref();
         return $verification->create($verifyLink);
     }
+
+//    public function update(array $array=[])
```
Comment (mjallday) [inline src/Finix/Resource.php]: Remove commented code

File: src/Finix/Hal/HrefSpec.php [hunk 3]
```diff
@@ -31,35 +31,35 @@ public function __construct($name, $idNames, $root = null, $override = null)
...
+//        // non-member
+//        if (count($parts) < count($this->idNames) + 1 ||
+//            $parts[count($parts) - 1 - count($this->idNames)] != $this->name
+//        ) {
+//            return null;
+//        }
+//
+//        // member
+//        $ids = array_combine(
+//            $this->idNames,
```
Comment (mjallday) [inline src/Finix/Hal/HrefSpec.php]: Remove comment d code

File: tests/Finix/Fixtures.php [hunk 4]
```diff
@@ -18,12 +18,12 @@
 
 class Fixtures extends \PHPUnit_Framework_TestCase
 {
-    public static $apiUrl = "https://api-staging.finix.io/";
+    public static $apiUrl = "https://api-test.payline.io";
```
Comment (mjallday) [inline tests/Finix/Fixtures.php]: this is wrong.

**Pull request merged**